### PR TITLE
[app_dart] Log ci.yaml exceptions

### PR DESCRIPTION
**Pull request by CaseyHillers**

### CODE REVIEW

**Review by christopherfujino — APPROVED** [review]
LGTM

File: app_dart/lib/src/service/scheduler.dart [hunk 1]
```diff
@@ -382,6 +382,7 @@ class Scheduler {
       );
     } else {
       log.warning('Marking PR #$prNumber ci.yaml validation as failed');
+      log.warning(exception.toString());
```
Comment (christopherfujino) [inline app_dart/lib/src/service/scheduler.dart:385]: looks like we already set this as the `github.CheckRunOutput()`

Comment (christopherfujino) [reply app_dart/lib/src/service/scheduler.dart:385]: You can capture the backtrace on the catches up on lines 365 and 368, like:

```
} catch (e, b) {
  log.warning(e.toString());
  log.warning(b.toString());
  exception = e;
  backtrace = b;
}
```

Comment (CaseyHillers) [reply app_dart/lib/src/service/scheduler.dart:386]: Thanks, done

File: app_dart/lib/src/service/luci.dart [hunk 2]
```diff
@@ -193,9 +193,10 @@ class LuciService {
     final BatchRequest batchRequest = BatchRequest(requests: searchRequests);
     final BatchResponse batchResponse = await buildBucketClient.batch(batchRequest);
     final Iterable<Build> builds = batchResponse.responses!
-        .map<SearchBuildsResponse?>((Response response) => response.searchBuilds)
-        .where((SearchBuildsResponse? response) => response?.builds != null)
-        .expand<Build>((SearchBuildsResponse? response) => response?.builds ?? <Build>[]) ?? <Build>[];
+            .map<SearchBuildsResponse?>((Response response) => response.searchBuilds)
+            .where((SearchBuildsResponse? response) => response?.builds != null)
+            .expand<Build>((SearchBuildsResponse? response) => response?.builds ?? <Build>[]) ??
+        <Build>[];
```
Comment (christopherfujino) [inline app_dart/lib/src/service/luci.dart:199]: is this really what the almighty dartfmt wants?

**Pull request merged**